freak/macros.xml

```diff
@@ -3,13 +3,15 @@
 <macros>
     <xml name="requirements">
         <requirements>
+            <requirement type="package" version="@VERSION@">freak</requirement>
+            <requirement type="package" version="0.23.4">pandas</requirement>
         </requirements>
     </xml>
 
     <xml name="version_cmd">
         <version_command>echo @VERSION@</version_command>
     </xml>
 
-    <token name="@VERSION@">0.1.0</token>
+    <token name="@VERSION@">0.1.1</token>
```

Comment:
Change back to 0.1.0; this version refers to the version of freak not the wrapper.